backend/users/views.py

```diff
@@ -0,0 +1,33 @@
+from django.http import HttpResponse,JsonResponse
```

Comment:
This file should be placed within a folder named viewsets and then this file should be named user_api_view.py